lib/assets/javascripts/turbograft/response.coffee

```diff
@@ -17,7 +17,17 @@ class TurboGraft.Response
     else
       throw new Error("Error encountered for XHR Response: #{this}")
 
+  redirectedTo: () ->
+    @_redirectedTo = @xhr.getResponseHeader('X-XHR-Redirected-To') unless @_redirectedTo
+    @_redirectedTo
```

Comment:
Hmm

Ok, so this was probably just over optimizing.

 I was intending to memoize the call to `getResponseHeader` onto the response object. That's why it's written this way (we can't just return the result of the assignment if we are only doing the assignment when it's not set...)

I think I'll just get rid of the memoization.